configs/STELLARF4/config.h

```diff
@@ -25,12 +25,17 @@
 #define BOARD_NAME STELLARF4
 #define MANUFACTURER_ID STBE
 
+#define TARGET_BOARD_IDENTIFIER "STF4"
+#define USBD_PRODUCT_STRING     "StingBee Stellar F4"
```

Comment:
nothing wrong with this, just noting that `src/platform/STM32/target/STM32F405/target.h` contains
```
#ifndef USBD_PRODUCT_STRING
#define USBD_PRODUCT_STRING     "Betaflight STM32F405"
#endif
```

Reply:
config.h is processed before target.h